package.json

```diff
@@ -22,7 +22,8 @@
     "test-stream": "npm run mocha test/stream.test.ts",
     "test-media": "npm run mocha test/media-upload.test.ts",
     "test-auth": "npm run mocha test/auth.test.ts",
-    "prepublish": "npm run build"
+    "prepublish": "npm run build",
+    "prepare": "npm run build"
```

Comment:
Thanks, 

Could be a good idea but I'm not convinced, I'm afraid it will make the CI build it twice, so let's remove it :)